### PR TITLE
added dog collar/leash

### PR DESCRIPTION
**Pull request by bleehu**
added collar and leash item.

### CODE REVIEW

File: Items/07_Items.txt [hunk 1]
```diff
@@ -43,6 +43,12 @@ from multiple angles to capture the whole object. Up to 20m^3 of space.
 Wearables
 ==============================
 
+Dog Collar ($45):
+    This 30cm wide collar comes with up to 18 characters of monogram and a micro
```
Comment (rosebrallier) [inline Items/07_Items.txt]: The width of a collar usually refers to the distance across the collar, not the length around; 30cm is really wide compared to the length of most dogs' necks.
In addition, current husbandry standards greatly favor harnesses over collars; harnesses don't choke a dog or cause spinal injuries when pressure is put on them.

**Review by rosebrallier — CHANGES_REQUESTED** [review]
Substitute a more reasonable collar width or swap to a harness altogether for realism and husbandry purposes.

File: Items/07_Items.txt (hunk 1 above)
Comment (bleehu) [reply Items/07_Items.txt]: Will change!

File: Items/07_Items.txt [hunk 2]
```diff
@@ -43,8 +43,8 @@ from multiple angles to capture the whole object. Up to 20m^3 of space.
 Wearables
 ==============================
 
-Dog Collar ($45):
-    This 30cm wide collar comes with up to 18 characters of monogram and a micro
+Dog Harness ($45):
+    This harness comes with up to 18 characters of monogram and a micro
 chip that helps you find your dog if it runs away. The collar itself is
```
Comment (rosebrallier) [inline Items/07_Items.txt]: Missed a reference to collar. Nb.

**Review by rosebrallier — APPROVED** [review]
Wooooooo!

**Pull request merged**